features/text_on_team_page.feature

```diff
@@ -0,0 +1,10 @@
+Feature: appropriate text on team page
+
+  As a concerned citizen
+  I want to know more about the people in this organization
+  So that I know what type of people I'm trusting
+
+Scenario: text on team page
+  Given I am on the home page
+  When I follow "Meet the Team"
+  Then I should see "Gresshaa Mehta"

```

Comment:
Maybe an and for the other teammates?

Reply:
Put some other stuff in, though since this page is hard-coded I don't really think this test does much. If there's too many team members or we decide to procedurally generate the bios I can be more descriptive here.